daktari/config.py

```diff
@@ -105,6 +107,33 @@ def parse_raw_config(config_path: Path, raw_config: str) -> Optional[Config]:
     return Config(min_version, title, checks)
 
 
+def is_python_version_on_path() -> bool:
+    path = (os.getenv("PATH") or "").lower()
+    try:
+        python3_version_raw = run_command(["python3", "--version"]).stdout.rstrip("\n")
+        python3_version = re.search("3\\.\\d+\\.\\d+", python3_version_raw)
+        if python3_version:
+            python3_version_minor = ".".join(python3_version.group().split(".")[0:2])
+        else:
+            return False
+    except CommandErrorException:
+        logging.debug("Could not run 'python3 --version'", exc_info=True)
+        return False
+
+    if re.search("python.?3", path):
+        if python3_version_minor in path:
+            logging.debug(f"Python 3 version is {python3_version_minor}. This was detected as being on the PATH.")
+            return True
+        else:
+            logging.debug(f"Python 3 version is {python3_version_minor}. This was NOT detected as being on the PATH.")
+            logging.debug("It is likely the terminal will be unable to locate a newly installed version of Daktari")
+            return False
+    else:
+        logging.debug("Python 3 does not appear to be on your PATH")
+        logging.debug("Assuming Daktari is install elsewhere in a non-versioned location")
```

Comment:
The check is specifically for Python, right? Which could be somewhere different from Daktari?

```suggestion
        logging.debug("Assuming Python is installed elsewhere in a non-versioned location")
```